src/layouts/HeaderContainer.tsx

```diff
@@ -96,7 +97,11 @@ class HeaderContainer extends React.Component<IProps, null> {
                     ? <div>
                         <AccountBalance tokenSymbol={dao.tokenSymbol} balance={member.tokens} accountAddress={web3State.ethAccountAddress} />
                         &nbsp; | &nbsp;
-                        <span>{member.reputation.toFixed(2).toLocaleString()} {dao.name} reputation</span>
+                        <Tooltip placement="bottom" trigger={["hover"]} overlay={<span style={{minHeight: 'inherit !important'}}>{member.reputation} reputation</span>}>
+                          <span style={{cursor: 'pointer'}}>
```

Comment:
a pointer usually means you can click on the element and have it do something, I dont think we want that here